src/main/java/org/archive/io/arc/ARCRecord.java

```diff
@@ -263,6 +264,7 @@ private ArchiveRecordHeader parseHeaders(final InputStream in,
             bodyOffset += getTokenizedHeaderLine(in, secondLineValues);
             version = ((String)secondLineValues.get(0) +
                 "." + (String)secondLineValues.get(1));
+            origin = (String)secondLineValues.get(2);
```

Comment:
Is it safe to assume there is always an Origin string at position 2? Or do we need to check secondLineValues.size() > 2?